### PR TITLE
Roll stats

### PR DESCRIPTION
**Pull request by egregors**
- [x] collection for hussars
- [x] document for hussar
- [x] store and API for hussar CRUD
- [x] update `roll` skill to store stats
- [x] base roll stats

rel #92 #93

### CODE REVIEW

File: bot/skills/roll.py [hunk 1]
```diff
@@ -15,10 +19,67 @@
...
+        })
+
+    def miss(self, user_id: str):
+        self._coll.update_one({"_id": user_id}, {
+            "$inc": {"shot_counter": 1, "miss_counter": 1},
+            "$set": {"last_shot": datetime.now()},
+        })
+
+
+_db = DB(db_name='roll')
```
Comment (C-Pro) [inline bot/skills/roll.py]: I think we should add option to erase own roll data to be GDPR compliant 🧐

File: bot/skills/roll.py [hunk 2]
```diff
@@ -15,10 +19,67 @@
 NUM_BULLETS = 6
 
 
+class DB:
+    """
+        Hussar document:
+        {
+            _id: 420,                                   # int       -- tg user id
+            meta: {...},                                # Dict      -- full tg user object (just in case)
```
Comment (C-Pro) [inline bot/skills/roll.py]: GDPR states that you can't store user-related data 'just in case', only when you actually need it 🧐

Comment (egregors) [reply bot/skills/roll.py]: This is all the data we are saving from the user. And all of them in public domain
```
        "id": {
            "$numberInt": "567937"
        },
        "first_name": "Egregors",
        "is_bot": false,
        "username": "egregors",
        "language_code": "en"
```

File: bot/skills/roll.py (hunk 1 above)
Comment (egregors) [reply bot/skills/roll.py]: `/roll_me_out` will remove all the data related to the current user

File: bot/skills/roll.py (hunk 2 above)
Comment (C-Pro) [reply bot/skills/roll.py]: Does not matter. It is 'his/her/its' data an 'he/she/it' is endowed with unalienable rights to withdraw them from our possession.

File: bot/skills/roll.py (hunk 1 above)
Comment (C-Pro) [reply bot/skills/roll.py]: Excellent! But I'd call it `/gdpr_me`

File: bot/skills/roll.py (hunk 2 above)
Comment (egregors) [reply bot/skills/roll.py]: sory I dont speack english

File: bot/skills/roll.py (hunk 1 above)
Comment (egregors) [reply bot/skills/roll.py]: Yep, like it

**Pull request merged**